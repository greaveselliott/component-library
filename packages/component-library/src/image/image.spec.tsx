import React from 'react';
import { shallow } from 'enzyme';

import Image from './image.component';

test('Renders correctly', () =>
  expect(
    shallow(<Image src="/" alt="A helpful description of this image." />),
  ).toMatchSnapshot());
