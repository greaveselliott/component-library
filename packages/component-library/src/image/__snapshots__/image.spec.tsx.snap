// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="image"
>
  <img
    alt="A helpful description of this image."
    className="image__tag"
    src="/"
  />
</div>
`;
